packages/feedback/src/sendFeedback.ts

```diff
@@ -0,0 +1,43 @@
+import type { BrowserClient, Replay } from '@sentry/browser';
+import { getCurrentHub } from '@sentry/core';
+import { getLocationHref } from '@sentry/utils';
+
+import type { SendFeedbackOptions } from './types';
+import { sendFeedbackRequest } from './util/sendFeedbackRequest';
+
+interface SendFeedbackParams {
+  message: string;
+  name?: string;
+  email?: string;
+  url?: string;
+}
+
+/**
+ * Public API to send a Feedback item to Sentry
+ */
+export function sendFeedback(
+  { name, email, message, url = getLocationHref() }: SendFeedbackParams,
+  { includeReplay = true }: SendFeedbackOptions = {},
+): ReturnType<typeof sendFeedbackRequest> {
+  const hub = getCurrentHub();
+  const client = hub && hub.getClient<BrowserClient>();
```

Comment:
```suggestion
  const client = getCurrentHub().getClient<BrowserClient>();
```

hub is always defined, no need to guard that!